src/System.Text.Json/tests/DebuggerTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Reflection;
+using Xunit;
+
+namespace System.Text.Json.Tests
+{
+    public class DebuggerTests
+    {
+        [Fact]
+        public void DefaultJsonElement()
+        {
+            // Validating that we don't throw on default
+            JsonElement element = default;
+            GetDebuggerDisplayProperty(element);
+        }
+
+        [Fact]
+        public void DefaultJsonProperty()
+        {
+            // Validating that we don't throw on default
+            JsonProperty property = default;
+            GetDebuggerDisplayProperty(property);
+        }
+
+        [Fact]
+        public void DefaultUtf8JsonWriter()
```

Comment:
Add test for reader as well?

Reply:
I can't as it is a ref struct (can't reflect into it).